.github/workflows/snyk.yml

```diff
@@ -0,0 +1,15 @@
+name: Snyk Monitor Scanning
+on:
+ release:
+   types: [published] 
+ push:
+    branches:
+      - 'main' 
+      - 'master'
```

Comment:
```suggestion
```
There is no `master` branch in this repository

Reply:
@andyasp Understood, this is a standard workflow file we are using across the Grafana Org. I included `master` just in case a repo still uses the old default branch naming.

Reply:
You may want to be careful about doing that since I'm unsure if every repository will necessarily have both branch protected. I've asked internally if `master` is here since it's currently unused.